javascript-loops-practice/get-student-names.js

```diff
@@ -0,0 +1,10 @@
+/* exported getStudentNames */
+function getStudentNames(students) {
+  var newArray = [];
+  for (var i = 0; i < students.length; i++) {
+    for (var student in students[i]) {
+      newArray.push(students[i][student]);
+    }
```

Comment:
You don't need this nested loop. You only need the value of the `name` property, and you know the specific name of that property, so you can just use dot notation to access that property